examples/matrix_multiplier/tests/Makefile

```diff
@@ -0,0 +1,79 @@
+TOPLEVEL_LANG ?= verilog
+SIM ?= icarus
+
+PWD=$(shell pwd)
+
+ifeq ($(OS),Msys)
+WPWD=$(shell sh -c 'pwd -W')
+else
+WPWD=$(shell pwd)
+endif
+
+
+# Matrix parameters
+DATA_WIDTH ?= 32
+A_ROWS ?= 10
+B_COLUMNS ?= 4
+A_COLUMNS_B_ROWS ?= 6
+
+ifeq ($(TOPLEVEL_LANG),verilog)
+    VERILOG_SOURCES = $(WPWD)/../hdl/matrix_multiplier.v
+
+    # Set module parameters
```

Comment:
For Cadence simulators, the option is ``-gpg`` for both parameters and generics, but I have to look up the exact formatting of the arguments.

Reply:
I found and added options for `ius`, but can't test them.